### ISSUE TITLE
[11.0] Initial login of the template user goes to Applications menu

### ISSUE THREAD

**Issue by BusinessTec** [it-projects-llc/odoo-saas-tools#802]
After creating the new database user is redirected to the newly create DB. In case the user is not allowed to access/add applications this is resulting in the error message. 
After accepting the message all seems to be working fine.

Is there a way to redirect user to another menu (I changed the user settings, but that seems no to be applicable to the initial session)?

**Comment by gokhanerdogdu** [comment]
Hi @BusinessTec 
Did you find a solution for this?

**Comment by BusinessTec** [comment]
Not really, looks like it is an Odoo 'feature' so I just created a landing promo page for the same model (read-only) to show the new users  other applications  as the premium products
